app/models/meter.rb

```diff
@@ -239,13 +239,23 @@ def active_meter_attributes
   def energy_tariff_meter_attributes
```

Comment:
This overrides the default implementation in energy tariff holder to do something slightly different. These are specs for this in meter spec. As you've modified this method, I think you also need to update the specs there to confirm that the `applies_to_meter_system` returns the right values and is used in this method correctly.